demo/components-demo.tsx

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
```

Comment:
General remarks:
1. The "down" icon is misplaced
2. The behavior is weird. IMHO when I start writing, the dropdown should open automatically if there are hits